tests/app-login.fg

```diff
@@ -0,0 +1,28 @@
+# Set up the database
+odin.sql.file (module.path.join ../Schema/core/000-initial.blue.sql)
+odin.sql.file (module.path.join ../Schema/authn/001-initial.blue.sql)
+odin.sql.file (module.path.join ../Schema/opts/app/002-initial.blue.sql)
+
+# Set up new users and apps
+odin.user owner password1234
+odin.user player1 password1234
+odin.user player2 password1234
+odin.sql.file (module.path.join app-login-test-figure.sql)
+
+GET odin/app /login/app01/ 200
+
+# Validate payload
+POST odin/app /login/app01/ {} 501
+POST odin/app /login/app01/app02/ {} 501
```

Comment:
I suspect that this should be a 404. There's a similar 404 case for where the app ID is missing isn't there?